src/app/layout.tsx

```
@@ -54,6 +54,18 @@ export default function RootLayout({
             `,
           }}
         />
+        {/* Google Analytics - Second Tracking ID */}
+        <script async src="https://www.googletagmanager.com/gtag/js?id=G-6RWEWQXJXD"></script>
+        <script
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)